build.assets/Dockerfile

```diff
@@ -159,6 +159,15 @@ RUN apt-get -y update && \
         xauth \
         libpcsclite-dev \
         && \
+    sudo install -m 0755 -d /etc/apt/keyrings && \
+    cat docker.gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg && \
+    sudo chmod a+r /etc/apt/keyrings/docker.gpg && \
+    echo \
+    "deb [arch="$(dpkg --print-architecture)" signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu \
+    "$(. /etc/os-release && echo "$VERSION_CODENAME")" stable" | \
+    sudo tee /etc/apt/sources.list.d/docker.list > /dev/null && \
+    sudo apt-get update && \
+    sudo apt-get install -y docker-ce-cli && \
```

Comment:
Can we add the repo before we do the big installation of all the packages, and just add `docker-ce-cli` to the list?

Reply:
Also, wouldn't it be simpler to just manually fetch and install the .deb, potentially pinning a version number for it too?

Reply:
> Can we add the repo before we do the big installation of all the packages, and just add docker-ce-cli to the list?

We need the gpg installed to load the docker signing cert and the docker repo requires ca-certificates to trust the TLS connection IIRC

>Also, wouldn't it be simpler to just manually fetch and install the .deb, potentially pinning a version number for it too?

I think  `docker-ce-cli` installs other packages besides the cli itself

Reply:
> Can we add the repo before we do the big installation of all the packages, and just add docker-ce-cli to the list?

We need the gpg installed to load the docker signing cert and the docker repo requires ca-certificates to trust the TLS connection IIRC

>Also, wouldn't it be simpler to just manually fetch and install the .deb, potentially pinning a version number for it too?

I think  `docker-ce-cli` installs other packages besides the cli itself